### ISSUE TITLE
EditMode Highlight disappears after editing component for the first time

### ISSUE THREAD

**Issue by dfosco** [vtex-apps/editor#21]
The banner element (small box with _olar_ alt text) should be highlighted in red here:

<img width="322" alt="erro-reopen-edit-mode" src="https://cloud.githubusercontent.com/assets/3229933/9137574/062d86c8-3cf5-11e5-919c-876c6aac2492.png">

**Comment by dfosco** [comment]
@bellacorbo We should start looking into these “old” bugs in the editor :smile:

**Comment by dfosco** [comment]
@brenoc Have you tested this to see if it's fixed?

**Comment by brenoc** [comment]
@dfosco No, but since we've changed a lot of the code related to this highlight, I'm closing it. If we see it again, we can reopen it.

**Comment by dfosco** [comment]
LGTM

2015-12-23 14:16 GMT-02:00 Breno Calazans <notifications@github.com>:

> @dfosco <https://github.com/dfosco> No, but since we've changed a lot of
> the code related to this highlight, I'm closing it. If we see it again, we
> can reopen it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vtex-apps/editor/issues/21#issuecomment-166934597>.
>



-- 
Daniel Fosco - UX
lab.vtex.com
+55 21 979571497
